tests/test_http.py

```diff
@@ -10,7 +10,7 @@
 
 from tests import config
 from tests.infra.server import ThreadedLocalServer, BaseHTTPRequestHandler
-from terra_notebook_utils.http import HTTPAdapter, Retry, http_session
+from terra_notebook_utils.http_session import HTTPAdapter, Retry, http_session
```

Comment:
I had to rename `http` file because of some naming conflict error

Reply:
What was the conflict?